### PR TITLE
Adds JDBC modules

### PR DESCRIPTION
**Pull request by chalkos**
* Adds JDBC import and export modules that work by including the jdbc driver jar file, specifying the name of the jdbc driver class and providing the connection URL
* Adds detailed instructions in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +127,40 @@ $ java -jar dbptk-app-x.y.z.jar \
 -e siard-1 -ef example.siard
 ```
 
+### How to use JDBC import and export modules
+
+To use Database Preservation Toolkit with an unsupported database, one can connect by providing the name of the the JDBC driver class (and adding the JDBC driver to the classpath) and the JDBC connection string. The steps to run Database Preservation Toolkit this way are as follows:
+
+1. Obtain Database Preservation Toolkit app;
```
Comment (luis100) [inline README.md]: This step is not necessary

File: README.md [hunk 2]
```diff
@@ -119,6 +127,40 @@ $ java -jar dbptk-app-x.y.z.jar \
...
+2. Obtain the JDBC driver for the database you want to use (this is typically a file with `jar` extension);
```
Comment (luis100) [inline README.md]: Give always an example, always the Oracle one (for now)

File: README.md [hunk 3]
```diff
@@ -119,6 +127,40 @@ $ java -jar dbptk-app-x.y.z.jar \
...
+3. Identify the driver class. For Oracle 12C this would be something like `oracle.jdbc.driver.OracleDriver`;
+4. Prepare the connection string. For Oracle 12C this could be something like `jdbc:oracle:thin:username/password@serverName:port/database`;
+5. Run Database Preservation Toolkit by providing files to add to the classpath and the main entry point.
+
+Please note that this method is likely to produce errors.
```
Comment (luis100) [inline README.md]: Replace with:
> Please be aware that using this method the conversion quality cannot be assured, as it depends on the used driver. Furthermore, non-tested drivers are more prone to possible errors during the conversion. A specialized module for the database, if available, would always be preferable to this generic JDBC module.

Comment (chalkos) [inline README.md]: much better :+1:

File: README.md [hunk 4]
```diff
@@ -119,6 +127,40 @@ $ java -jar dbptk-app-x.y.z.jar \
...
+
+#### Example to convert from Oracle to SIARD2:
+
+Assuming that all the files have been obtained, on Windows the command to start Database Preservation Toolkit could be something like:
```
Comment (luis100) [inline README.md]: > Using the method described above, the Windows command to extract a database from an Oracle database to SIARD 2 is as the following:

**Comment by chalkos** [comment]
Fixes #81

**Pull request merged**